charts/karmada/templates/karmada-scheduler.yaml

```
@@ -60,6 +60,10 @@ spec:
             initialDelaySeconds: 15
             periodSeconds: 15
             timeoutSeconds: 5
+          ports:
+            - containerPort: 10351
+              name: http
```

```suggestion
              name: metrics
```
Why not name it with `metrics`?

---

No, let all metrics with 8080 would be fine.
Ignore this comment.